README.md

```diff
@@ -108,4 +108,4 @@ Feel free to [create an issue](https://github.com/0xType/0xProto/issues?q=is%3Ai
 
 The SIL Open Font License, Version 1.1
 
-Copyright (c) 2025 [0xType](https://0xtype.dev)
\ No newline at end of file
+Copyright (c) 2025 [0xType](https://0xtype.dev)
```

Comment:
This line does not need to be changed, so can you please undo the change?

Reply:
I did not touch it. I tested on my fork and it did the same thing without me touching that line.
![Screenshot-2025-05-29-12-33-17](https://github.com/user-attachments/assets/1884ee57-9697-4586-aa08-8c145fbf7ec7)

Reply:
It appears that a newline code was added, but it's OK

<img width="1215" alt="image" src="https://github.com/user-attachments/assets/e0f8e9b9-4934-4ede-9ffe-d8644f45f8c5" />

Reply:
I don't know what to say. I used GitHub's web editor and only edited the Download URL.